app/src/main/java/org/mozilla/focus/utils/Features.kt

```diff
@@ -9,9 +9,9 @@ package org.mozilla.focus.utils
  */
 object Features {
     const val SEARCH_TERMS_OR_URL: Boolean = true
-    val SHOULD_SHOW_CFR_FOR_SHIELD_TOOLBAR_ICON = AppConstants.isDevOrNightlyBuild
-    val SHOULD_SHOW_TOOLTIP_FOR_PRIVACY_SECURITY_SETTINGS_SCREEN = AppConstants.isDevOrNightlyBuild
-    const val SHOULD_SHOW_HOME_PAGE_PRO_TIPS = true
+    val SHOW_TRACKING_PROTECTION_CFR = AppConstants.isDevOrNightlyBuild
+    val SHOW_TOOLTIP_FOR_PRIVACY_SECURITY_SETTINGS_SCREEN = AppConstants.isDevOrNightlyBuild
+    const val SHOW_HOME_PAGE_PRO_TIPS = true
```

Comment:
Not sure why these renamings were necessary?
Show seem imperative to me like something we need to do - but there are other checks that might prevent that so even though this says "show" we might not "show" a certain CFR.
If we are to change the namings and set an example for the future maybe we could use a `is<FeatureName>Enabled` form?